### PR TITLE
Fix minor things in project

### PR DESCRIPTION
**Pull request by AbeeraTahir**
In this PR, I have:

- Removed unused code
- Modified output pattern for music albums and games
- Fixed bugs in genre and author modules

### CODE REVIEW

File: src/app.rb [hunk 1]
```diff
@@ -23,16 +23,14 @@ def initialize
...
-    10 => :add_a_music_album,
-    11 => :add_a_movie,
-    12 => :add_a_game,
+    3 => :list_all_games,
+    4 => :list_all_genres,
+    5 => :list_labels,
+    6 => :list_all_authors,
+    7 => :add_a_book,
+    8 => :add_a_music_album,
+    9 => :add_a_movie,
```
Comment (jellywiz) [inline src/app.rb]: we dont have this as well

File: src/app.rb [hunk 2]
```diff
@@ -41,16 +39,14 @@ def show_interactive_console
...
-    puts '10. Add a music album'
-    puts '11. Add a movie'
-    puts '12. Add a game'
+    puts '3. List all games'
+    puts '4. List all genres'
+    puts '5. List all labels'
+    puts '6. List all authors'
+    puts '7. Add a book'
+    puts '8. Add a music album'
+    puts '9. Add a movie'
```
Comment (jellywiz) [inline src/app.rb]: here as well

**Review by jellywiz — CHANGES_REQUESTED** [review]
please check the comments below

File: src/app.rb (hunk 1 above)
Comment (AbeeraTahir) [reply src/app.rb]: Thanks for highlighting! I will resolve it.

File: src/app.rb (hunk 2 above)
Comment (AbeeraTahir) [reply src/app.rb]: Thanks for highlighting! I will resolve it.

Comment (dialite) [reply src/app.rb]: Thanks for the observation, some are meant to be for a fourth person which our team does not have.

**Review by dialite — APPROVED** [review]
# Approved :green_circle: 
Hi @AbeeraTahir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Properly removed codes not needed :white_check_mark: 
- [ ] GitHub flow is correctly used :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Good descriptive pull request title and summary :white_check_mark: 

Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**